src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1126,7 +1126,7 @@ public override string VisualizeLocalType(object type)
 
         #region IOperation tree validation
 
-        protected static (IOperation operation, SyntaxNode node) GetOperationAndSyntaxForTest<TSyntaxNode>(CSharpCompilation compilation)
+        protected static (IOperation operation, SyntaxNode node) GetOperationAndSyntaxForTest<TSyntaxNode>(CSharpCompilation compilation, bool highLevelOperation = true)
```

Comment:
Nit: I'd prefer if the flag is false by default, perhaps `useLoweredTree`?

Reply:
It also probably shouldn't be defaulted for this method, only for the methods that are actually used in the tests. That way we make sure we're actually passing it everywhere that matters.